### PR TITLE
chore: Separate out manual/generated code in the BigQuery generator

### PR DESCRIPTION
**Pull request by jskeet**
This was effectively in response to a question from Shiv about what code was generated - I realized that the mixture here was unhelpful.

### CODE REVIEW

File: apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Program.cs [hunk 1]
```diff
@@ -53,12 +51,15 @@ private static int Main(string[] args)
...
             {
                 XDocument document = XDocument.Load(args[0]);
-                var code = File.ReadLines(args[1]);
-                foreach (var methodElement in document.Root.Elements("Method"))
-                {
-                    ApiMethod method = new ApiMethod(methodElement);
-                    code = ModifyCode(method, code);
-                }
+                var methods = document.Root.Elements("Method").Select(element => new ApiMethod(element));
+                var methodCode = methods.SelectMany(method => method.GenerateCode()).ToList();
```
Comment (Rishabh-V) [inline apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Program.cs:55]: super nit: methodCode => allMethodsCode or any other plural name?
(I saw `ToList()` and inferred that this variable holds code for all the methods and then got confused if it's just one method code based on the variable name)

**Review by Rishabh-V — APPROVED** [review]
This looks great. Thank you very much Jon.

File: apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Program.cs (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Program.cs:55]: Sure, happy to do so.

Comment (jskeet) [reply apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Program.cs]: Done.

**Pull request merged**